kyuubi-ha/src/test/scala/org/apache/kyuubi/ha/client/etcd/EtcdDiscoveryClientSuite.scala

```diff
@@ -31,20 +30,26 @@ import org.apache.kyuubi.ha.client.DiscoveryClientTests
 import org.apache.kyuubi.service.NoopTBinaryFrontendServer
 
 class EtcdDiscoveryClientSuite extends DiscoveryClientTests {
+
+  val etcdVersion = "3.5.4"
+
   private var etcdCluster: EtcdCluster = _
   var engineServer: NoopTBinaryFrontendServer = _
 
-  private lazy val _connectString = etcdCluster.clientEndpoints().asScala.mkString(",")
+  private lazy val _connectString = etcdCluster.clientEndpoints.asScala.mkString(",")
 
-  override def getConnectString(): String = _connectString
+  override def getConnectString: String = _connectString
 
   val conf: KyuubiConf = {
     KyuubiConf()
       .set(HA_CLIENT_CLASS, classOf[EtcdDiscoveryClient].getName)
   }
 
   override def beforeAll(): Unit = {
-    etcdCluster = new Etcd.Builder().withNodes(2).build()
+    etcdCluster = new Etcd.Builder()
+      .withImage(s"gcr.io/etcd-development/etcd:v$etcdVersion")
```

Comment:
shall we also change those `builder`
https://github.com/apache/incubator-kyuubi/blob/87782097976fd7732ad0b455613369b77ea78bad/externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/WithEtcdCluster.scala#L40-L44
https://github.com/apache/incubator-kyuubi/blob/87782097976fd7732ad0b455613369b77ea78bad/kyuubi-server/src/test/scala/org/apache/kyuubi/engine/EngineRefWithEtcdSuite.scala#L42-L46

Reply:
Fixed